src/main/java/com/incentives/piggyback/offers/entity/OfferEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.incentives.piggyback.offers.entity;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.incentives.piggyback.offers.dto.Location;
+
+import lombok.Data;
+
+@Document(collection = "offer")
+@Data
+public class OfferEntity {
+
+	@Id
+	private Long offerId;
+	private String orderId;
+	private String partnerId;
+	private String vendorName;
+	private String vendorAppUrl;
```

Comment:
What are vendorName and vendorAppUrl? If these belong to the partner, they should be named partnerName and partnerAppUrl

Reply:
done